menas/src/test/resources/application.properties

```diff
@@ -56,7 +56,8 @@ menas.hadoop.auth.krb5.keytab=hdfs.keytab
 menas.auth.inmemory.user=user
 menas.auth.inmemory.password=chang< )eme
 
-menas.mongo.connection.string=mongodb://localhost:27017
+# embedded mongo for integTest will be launched instead on a random free port
+menas.mongo.connection.string=mongodb://not-used-for-testing:12345
```

Comment:
This cannot be currently removed, because the `defaultMongoDb` still requires it (e.g. for the theoretical case where would want to run a test without the embedded Mongo).

Reply:
It's okay, since it is inside *test* resources.